src/CoreBundle/Entity/UserRelTag.php

```diff
@@ -20,13 +20,35 @@
  */
 class UserRelTag
 {
+//    /**
+//     * @var int
+//     *
+//     * @ORM\Column(name="user_id", type="integer", nullable=false)
+//     */
+//    protected $userId;
     /**
-     * @var int
+     * @ORM\OneToOne (targetEntity="Chamilo\CoreBundle\Entity\User",
+     *      inversedBy="user_rel_tag")
```

Comment:
`user_rel_tag` is incorrect for `inversedBy`. Because it should be the class property in User entity.